test/custom-options.test.ts

```diff
@@ -1,13 +1,15 @@
-const gotScraping = require('../src');
-const { startDummyServer } = require('./helpers/dummy-server');
+import { Server } from 'http';
+import { AddressInfo } from 'net';
+import gotScraping, { OptionsInit } from '../src';
```

Comment:
i believe the linter errors should go away too when you update `@apify/eslint-config-ts` to latest